### ISSUE TITLE
Assets not precompiled

### ISSUE THREAD

**Issue by sepastian** [resque/resque-web#106]
Hello,

I just upgraded to `0.0.8` and now cannot access the web interface anymore. I get:

```
Sprockets::Rails::Helper::AssetNotPrecompiled in ResqueWeb::Overview#show

Showing /home/sebastian/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/resque-web-0.0.8/app/views/layouts/resque_web/application.html.erb where line #41 raised:

Asset was not declared to be precompiled in production.
Add `Rails.application.config.assets.precompile += %w( resque_web/lifebuoy.png )` to `config/initializers/assets.rb` and restart your server
```

After extending `config/initializers/assets.rb` with:

```
Rails.application.config.assets.precompile += %w(resque_web/*.png)
```

everything works as expected.

What's wrong? Shouldn't the line above be part of `resque-web` somehow?

**Comment by mcfiredrill** [comment]
I'm guessing we need to add that line to resque-web to be compatible with newer versions of rails, hopefully that is backwards compatible to add it.

**Comment by chrishough** [comment]
+1, I just had the same issue

**Comment by ah-tunecore** [comment]
+1, I also had this issue.  Adding the line to `initializers/assets.rb` fixed it for now...

**Comment by kirillplatonov** [comment]
Fixed in #112

**Comment by bitterloa** [comment]
FYI - I was having the same error occur about the assets not being in the pipeline, however in my case it was because I am using the newest version of Sprockets 4 in my Rails app. I no longer use `initializers/assets.rb` and instead use the simpler `app/assets/config/manifest.js` file to manage my assets. I love it! But, when I upgraded Sprockets to 4 and manifest.js I needed to add the following files to make it work:

```
/* 
  Below are Assets for the resque-web gem > website.com/resque_web 
*/
//= link resque_web/application.js
//= link resque_web/application.css
//= link resque_web/idle.png
//= link resque_web/lifebuoy.png
//= link resque_web/poll.png
//= link resque_web/rails.png
//= link resque_web/working.png
```

Now it works (again)!
